zotbins/Waste-Watcher-AWS-IoT-Core
# Created MQTT test scripts

## Description

Created an MQTT test script in order to verify that the MQTT messages from the Waste Watcher are correct and valid. 

## Type of change


## Testing and Verification Instructions

1. Follow the instructions on the README to set up the MQTT scripts. 
2. Run the ```mqtt_sub.py``` and then run ```mqtt_pub.py``` to send the MQTT messages.
3. Verify that the image was saved correctly and the fullness was printed to the terminal. 

## Checklist:
